### PR TITLE
Fixed tests for basic_gnn_edgecnn and basic_gnn_gcn

### LINKED ISSUE #8103: Model support for `basic_gnn_edgecnn` with Torch_XLA2
## Fix the model test for `basic_gnn_edgecnn.py`

1. setup env according to [Run a model under torch_xla2](https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/docs/support_a_new_model.md)
2. Run model test under `run_torchbench/` with `python models/your_target_model_name.py`
3. Fix the failure.

Please refer to 
[this guide](https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/docs/support_a_new_model.md) as guide to fix:

Also refer to these PRs:
* https://github.com/pytorch/xla/pull/7583

### PR DESCRIPTION
**Pull request by anishfish2**
Fixes #8103 and #8104 by adding @op decorators to allow the tests to pass.

### CODE REVIEW

File: experimental/torch_xla2/torch_xla2/ops/jaten.py [hunk 1]
```diff
@@ -1485,6 +1485,7 @@ def _scatter_index(dim, index):
 
 
 # aten.scatter_add
+@op(torch.ops.aten.scatter_add_)
```
Comment (qihqi) [inline experimental/torch_xla2/torch_xla2/ops/jaten.py:1488]: for ops that ends with _ ; those are usually mutation ops (ops that both return the result as well as modify the first input). Let's add an entry here: https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/torch_xla2/ops/jaten.py#L49 instead.

File: experimental/torch_xla2/torch_xla2/ops/jaten.py [hunk 2]
```diff
@@ -1567,6 +1568,7 @@ def _aten_atan(self):
 # aten.scatter_reduce
 @op(torch.ops.aten.scatter)
 @op(torch.ops.aten.scatter_reduce)
+@op(torch.ops.aten.scatter_reduce_.two)
```
Comment (qihqi) [inline experimental/torch_xla2/torch_xla2/ops/jaten.py:1571]: same here.

File: experimental/torch_xla2/torch_xla2/ops/jaten.py (hunk 1 above)
Comment (anishfish2) [reply experimental/torch_xla2/torch_xla2/ops/jaten.py]: Done.

File: experimental/torch_xla2/torch_xla2/ops/jaten.py (hunk 2 above)
Comment (anishfish2) [reply experimental/torch_xla2/torch_xla2/ops/jaten.py]: Done.

**Comment by Abhishek-More** [comment]
Great work!

**Pull request merged**